doc/classes/VisualShaderNodeCustom.xml

```diff
@@ -0,0 +1,138 @@
+			</description>
+		</method>
+		<method name="_get_input_port_type" qualifiers="virtual">
+			<return type="int" enum="VisualShaderNode.PortType">
+			</return>
+			<argument index="0" name="port" type="int">
+			</argument>
+			<description>
+				By overriding this function, the returned type (see [constant VisualShaderNode.PortType]) will be used for associated input [code]port[/code].
+				Does not required to be overrided (but usually desired). If its not overrided, all input ports will return [constant VisualShaderNode.PORT_TYPE_SCALAR] type.
+			</description>
+		</method>
+		<method name="_get_name" qualifiers="virtual">
+			<return type="String">
+			</return>
+			<description>
+				By overriding this function, name of the associated custom node in the members dialog (and graph) will be changed to the returned value.
+				Does not required to be overrided (but usually desired). If its not overrided, the custom node will be named as "Unnamed".
+			</description>
+		</method>
+		<method name="_get_output_port_count" qualifiers="virtual">
+			<return type="int">
+			</return>
+			<description>
+				By overriding this function, the output port amount is changed to the returned value.
+				Required to be overrided in most cases (by default, no output ports are exist).
+			</description>
+		</method>
+		<method name="_get_output_port_name" qualifiers="virtual">
+			<return type="String">
+			</return>
+			<argument index="0" name="port" type="int">
+			</argument>
+			<description>
+				By overriding this function, the returned string will be used as a name for associated output [code]port[/code].
+				Does not required to be overrided (but usually desired). If its not overrided, all output ports will get names: [code]out + portIndex[/code].
+			</description>
+		</method>
+		<method name="_get_output_port_type" qualifiers="virtual">
+			<return type="int" enum="VisualShaderNode.PortType">
+			</return>
+			<argument index="0" name="port" type="int">
+			</argument>
+			<description>
+				By overriding this function, the returned type (see [constant VisualShaderNode.PortType]) will be used for associated output [code]port[/code].
+				Does not required to be overrided (but usually desired). If its not overrided, all output ports will return [constant VisualShaderNode.PORT_TYPE_SCALAR] type.
+			</description>
+		</method>
+		<method name="_get_return_icon_type" qualifiers="virtual">
+			<return type="int" enum="VisualShaderNode.PortType">
+			</return>
+			<description>
+				By overriding this function, return icon of the associated custom node in the members dialog will be changed to the returned type.
+				Does not required to be overrided.
+			</description>
+		</method>
+		<method name="_get_sub_category" qualifiers="virtual">
+			<return type="String">
+			</return>
+			<description>
+			By overriding this function, category of the associated custom node in the members dialog will be changed to the returned value.
```

Comment:
Indentation is wrong.

Proofreading:
```
Override this method to define the subcategory of the associated custom node in the Visual Shader Editor's members dialog.
Defining this method is [b]optional[/b]. If not overridden, the node will be filed under the root of the main category (see [method _get_category]).
```

Reply:
I'm not familiar with the VSE yet so it's not clear to me what subcategories are, so I hope I got things right in my reformulation.
May also be worth renaming the method to `_get_subcategory`? I think "subcategory" is more common than "sub category".

Reply:
Ok..